cmd/apps.go

```diff
@@ -50,7 +50,7 @@ func (d DeisCmd) AppCreate(id, buildpack, remote string, noRemote bool) error {
 
 	if !noRemote {
 		if err = git.CreateRemote(s.Client.ControllerURL.Host, remote, app.ID); err != nil {
-			if err.Error() == "exit status 128" {
+			if strings.Contains(err.Error(), "Error when running 'git remote add") {
```

Comment:
Good idea. This actually caught a bug in the unit tests I'm writing too. It was catching `[Error when running 'git remote add deis ssh://git@127-builder.0.0.1:2222/foo.git'
       		fatal: Not a git repository (or any of the parent directories): .git` which is of course an error we did not intend to catch.